src/firmware.c

```diff
@@ -25,11 +27,34 @@
 
 uint32_t __stack_chk_guard = 0;
 
+/**
+ * Sets the correct memory configuration for this firmware.
+ *
+ * Extends the APPDATA_1 MPU region up to the end of the
+ * flash memory, and sets it to R/W.
+ *
+ * FUTURE: maybe add SmartEEPROM support here?
```

Comment:
remove this?

Reply:
That will come with https://github.com/digitalbitbox/bitbox02-firmware/pull/205

Reply:
Removed, as this has been moved to `mpu.c`